### DISCUSSION TITLE
How to construct a Face_range?

### DISCUSSION THREAD

**Discussion by StromanStroman** [CGAL/cgal#7331] [Q&A] [answered]
I just to how to get a face_range,but don't know to create it.
Will anyone tell me how to do that?

**Comment by afabri** [comment, marked as answer]
A `std::vector<boost::graph_traits<Surface_mesh<Point>>::face_descriptor>`   is an example for a face range.

In the documentation of [`triangulate_faces()`](https://doc.cgal.org/latest/Polygon_mesh_processing/group__PMP__meshing__grp.html#gaf094a938993d7353589a55010edf63a6)   you should follow the link for `Range`  to see what it means more generally.
